domino_environments/_environments.py

```diff
@@ -235,11 +252,16 @@ def create_environment(
         if cluster_types:
             form_payload["clusterTypes[]"] = cluster_types
 
-        return self.request_manager.post(
+        response = self.request_manager.post(
             url=self._routes.environment_create(),
             data=form_payload,
             headers={"Content-Type": "application/x-www-form-urlencoded"},
         )
+        if response.ok:
+            return (response, response.url.split("/")[4])
+        else:
+            # Should we maybe raise an exception here?
+            return (response, None)
```

Comment:
Yes, instead of returning the response, we can raise an error depending on the status.
https://docs.python-requests.org/en/master/user/quickstart/#response-status-codes

```suggestion
            response.raise_for_status()
            return (response, None)
```

When we do `raise_for_status()` if the `status_code` is OK or any other non-error related code then it will pass without an issue.